src/dos/dos_programs.cpp

```diff
@@ -96,10 +99,23 @@ static const char* UnmountHelper(char umount) {
 
 class MOUNT : public Program {
 public:
-	void Move_Z(char new_z) {
-		char newz_drive = (char) toupper(new_z);
-		int i_newz = drive_index(newz_drive);
-		if (i_newz >= 0 && i_newz < DOS_DRIVES-1 && !Drives[i_newz]) {
+	void Move_Z(char new_z)
```

Comment:
Only comment here is the (admittedly petty) inconsistency between how the `z` is delineated from or attached to `new`, move`, and `drive` words:
Move_Z
moveZ
New_z
Newz_
Newz_drive
Zdrive_num

I specifically struggle with "newz" because I have anti-read it (not to see it as news with a zee). 

Maybe easiest just to always separate the _z_, which should make it easier to read too.

Reply:
I agree - made some modifications:

- `i_newz` -> `new_idx`
- `newz_drive` -> `new_drive_z`
- newly introduced `PROGRAM_MOUNT_MOVEZ_ERROR_1` -> `PROGRAM_MOUNT_MOVE_Z_ERROR_1`

Reply:
**Newz** becoming `..._idx`, and `..._drive_z`. .. are sooo much easier to quickly read (I'm not feeling so petty).  Thank you!